### PR TITLE
chore: bring back canonical urls

### PR DESCRIPTION
**Pull request by pankajagrawal16**

### CODE REVIEW

File: docs/aca/07-aca-cron-bindings/index.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+canonical_url: https://bitoftech.net/2022/09/09/azure-container-apps-monitoring-and-observability-with-application-insights-part-8/
```
Comment (tjoudeh) [inline docs/aca/07-aca-cron-bindings/index.md:2]: this should be: https://bitoftech.net/2022/09/05/azure-container-apps-with-dapr-bindings-building-block/

File: docs/aca/08-aca-monitoring/index.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+---
+canonical_url: https://bitoftech.net/2022/09/22/azure-container-apps-auto-scaling-with-keda-part-11/
```
Comment (tjoudeh) [inline docs/aca/08-aca-monitoring/index.md:2]: this should be: https://bitoftech.net/2022/09/09/azure-container-apps-monitoring-and-observability-with-application-insights-part-8/

File: docs/aca/09-aca-autoscale-keda/index.md [hunk 3]
```diff
@@ -1,9 +1,7 @@
 ---
-title: Module 9 - ACA Auto Scaling with KEDA
-has_children: false
-nav_order: 9
-canonical_url: 'https://bitoftech.net/2022/09/22/azure-container-apps-auto-scaling-with-keda-part-11/'
+canonical_url: https://bitoftech.net/2022/09/16/use-bicep-to-deploy-dapr-microservices-apps-to-azure-container-apps-part-10/
```
Comment (tjoudeh) [inline docs/aca/09-aca-autoscale-keda/index.md:2]: this should be: https://bitoftech.net/2022/09/22/azure-container-apps-auto-scaling-with-keda-part-11/

**Review by tjoudeh — COMMENTED** [review]
@pankajagrawal16 thank you for the changes, please check my comments as some URLs on the modules are incorrect, you need just to update the URL value as per the comment. Please make the changes and complete the PR.

File: docs/aca/07-aca-cron-bindings/index.md (hunk 1 above)
Comment (pankajagrawal16) [reply docs/aca/07-aca-cron-bindings/index.md:2]: Interesting. It was what you shared in the PR. But anyways i will fix. Thanks for spotting

**Pull request merged**